assets/locales/nl.json

```diff
@@ -49,7 +49,7 @@
       "buttonTitleInfected": "Ik denk dat ik besmet ben"
     },
     "endangerment": {
-      "info": "Vertel of je symptomen heeft of een\npositieve Covid-19 test resultaat hebt",
+      "info": "Vertel of je symptomen hebt of een\npositieve Covid-19 test resultaat hebt",
```

Comment:
"info": "Vertel of je symptomen hebt of een\npositief Covid-19 test resultaat hebt",